src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.tsx

```diff
@@ -63,61 +58,59 @@ export const KafkaDetailsTab: FunctionComponent<KafkaDetailsTabProps> = ({
 
   return (
     <div className="mas--details__drawer--tab-content">
-      <DetailsTabAlert expiryDate={expiryDate} />
+      {instanceType !== "standard" && expiryDate && (
```

Comment:
I like how @riccardo-forina done creation screen setting up some states as input logic.
Maybe it could work here as well.

States like: hasExpiration etc.

Reply:
I'm not aware about hasExpiration. Can you share link where we are using hasExpiration ?

Reply:
```suggestion
      {expiryDate && (
```

I'd keep the logic tied to the presence of the `expiryDate`. Checking for the instance type doesn't add much value imho.